rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/condition/Version.java

```diff
@@ -0,0 +1,96 @@
+package org.jboss.windup.rules.apps.java.archives.condition;
+
+/**
+ * Object used to specify the version range
+ *
+ */
+public class Version
+{
+    private String from;
+    private String to;
+
+    public static Version fromVersion(String from)
+    {
+        Version v = new Version();
+        v.setFrom(from);
+        return v;
+    }
+
+    public static Version toVersion(String to)
+    {
+        Version v = new Version();
+        v.setTo(to);
+        return v;
+    }
+
+    public Version to(String to)
+    {
+        this.setTo(to);
+        return this;
+    }
+
+    public String getFrom()
+    {
+        return from;
+    }
+
+    public void setFrom(String from)
+    {
+        this.from = from;
+    }
+
+    public String getTo()
+    {
+        return to;
+    }
+
+    public void setTo(String to)
+    {
+        this.to = to;
+    }
+
+    public boolean validate(String versionString)
+    {
+        boolean result = true;
+        if (from != null)
+        {
+            result = result && firstVersionLesser(from, versionString);
+        }
+        if (to != null)
+        {
+            result = result && firstVersionLesser(versionString, to);
+        }
+        return result;
+    }
```

Comment:
This would be quicker if we returned false straight away if the first test failed rather than then running the second one anyway

Reply:
Again a very good suggestion :+1: